umap/static/umap/js/modules/data/layer.js

```diff
@@ -61,6 +61,13 @@ export class DataLayer {
     this._isDirty = false
     this._isDeleted = false
     this.setUmapId(data.id)
+
+    if (!this.umap_id) {
```

Comment:
Why don't we reuse `createdOnServer` here?

Reply:
We now can, this is actually a leftover from a previous iteration, where I couldn't actually rely on `createdOnServer`. Let's change this!